app/javascript/widget/components/ArticleCardSkeletonLoader.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <div class="py-4 space-y-4 bg-white">
+    <div class="space-y-2 animate-pulse ">
+      <div class="h-6 bg-slate-100 rounded w-2/5" />
+    </div>
+    <div class="space-y-2 animate-pulse ">
+      <div class="h-4 bg-slate-100 rounded" />
+      <div class="h-4 bg-slate-100 rounded" />
+      <div class="h-4 bg-slate-100 rounded" />
+    </div>
+    <div class="space-y-2 animate-pulse ">
+      <div class="h-4 bg-slate-100 rounded w-1/5" />
+    </div>
```

Comment:
The animating color is white in dark mode.

<img width="401" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/c913284e-6fd1-44e1-b451-5cfad8c00da6">


```suggestion
    <div class="space-y-2 animate-pulse ">
      <div class="h-6 bg-slate-100 dark:bg-slate-500 rounded w-2/5" />
    </div>
    <div class="space-y-2 animate-pulse ">
      <div class="h-4 bg-slate-100 dark:bg-slate-500 rounded" />
      <div class="h-4 bg-slate-100 dark:bg-slate-500 rounded" />
      <div class="h-4 bg-slate-100 dark:bg-slate-500 rounded" />
    </div>
    <div class="space-y-2 animate-pulse ">
      <div class="h-4 bg-slate-100 dark:bg-slate-500 rounded w-1/5" />
    </div>
```